scripts/js_libraries/README.md

```diff
@@ -0,0 +1,18 @@
+# Scripts
+
+## get_retirejs_db.py
+
+Running this script retrieves vulnerability information from various javascript libraries from
+[retire.js](https://github.com/RetireJS/retire.js/blob/master/repository/jsrepository-master.json)
+and only keeps the contents under each library's "extractors" section, which contains regexes
+relevant for detecting a specific javascript library by its file name, file contents, or md5 hash.
+
+The resulting smaller json is written to `reduced.json` in the same directory. This smaller file
+will be read from to make the checks later on.
```

Comment:
Since `js_libraries.regex` is a JSON file, maybe we should rename it in a way that has a json extension (e.g. `js_library_patterns.json` or `js_library_regexes_db.json`). And then update any references to the new file name.